mongo/change_stream.go

```diff
@@ -101,6 +101,55 @@ type changeStreamConfig struct {
 	crypt          driver.Crypt
 }
 
+// mergeChangeStreamOptions combines the given ChangeStreamOptions instances into a single ChangeStreamOptions in a
+// last-one-wins fashion.
```

Comment:
Optional: what do you think of this change? It emphasizes that it's not the last object wins, but rather the last property defined for each object.
```suggestion
// last-property-wins fashion.
```